projects/demo/src/modules/components/surface/index.html

```diff
@@ -24,21 +18,22 @@
         <tui-doc-example
             id="behaviors"
             heading="Behaviors"
-            [content]="example1"
+            [component]="1 | tuiComponent"
+            [content]="1 | tuiExample: 'html,less'"
         >
             <tui-notification class="tui-space_bottom-4">
                 You can enable hover effects only on devices with pointer:
                 <div>
                     <code>&#64;media (hover: hover) and (pointer: fine)</code>
                 </div>
             </tui-notification>
-            <tui-surface-example-1 />
         </tui-doc-example>
 
         <tui-doc-example
             id="presets"
             heading="Presets"
-            [content]="example2"
+            [component]="2 | tuiComponent"
+            [content]="2 | tuiExample: 'html'"
```

Comment:
Why not go over them with ngFor? We need typescript all the time anyway, since these are standalone components that include imports.